python/packages/autogen-ext/src/autogen_ext/models/llama_cpp/_llama_cpp_completion_client.py

```diff
@@ -90,28 +90,33 @@ async def create(
                 converted_messages.append({"role": "user", "content": msg.content})
             elif isinstance(msg, AssistantMessage) and isinstance(msg.content, str):
                 converted_messages.append({"role": "assistant", "content": msg.content})
+            elif (
+                isinstance(msg, SystemMessage) or isinstance(msg, UserMessage) or isinstance(msg, AssistantMessage)
+            ) and isinstance(msg.content, list):
+                raise ValueError("Image messages are not currently supported.")
```

Comment:
Nit: Technically speaking there may not be an image in the list. Better error message: "Multi-part messages such as those containing images are currently not supported."

Reply:
Done